web/app/api/downloads_info.py

```diff
@@ -67,7 +67,8 @@ def get_credentials(body: GetCredentials):
 
     if computer:
         print("computer: ", computer, computer.computer_name)
-        # computer.last_time_online = CFG.offset_to_est(datetime.datetime.now())  # TODO couses error in tests but works ok on server
+        # TODO couses error in tests but works ok on server
+        # computer.last_time_online = CFG.offset_to_est(datetime.datetime.now())
         computer.last_time_online = CFG.offset_to_est(datetime.datetime.now(), True)

```

Comment:
@Taraslysak 
Timezone awareness requires additional package to install. Hello, Python. And this will result in bigger .exe and .msi files sizes. So, decision was made to use native tools and make it simple, dropping the elegance.
This whole project is for EST timezone only. The question was asked directly and client replied that no more timezones are required.
Well, time shifting is working respectfully to UTC standard. No concerns were raised at this point